Jenkinsfile

```diff
@@ -91,7 +91,7 @@ def runTests = { Map settings ->
             --network ${testNetwork} \\
             --volumes-from ${dindContainerName} \\
             ${testImage} \\
-            py.test -v -rxs tests/integration
+            py.test -vv -rxs tests/integration
```

Comment:
I don't mind reverting, but I think it can be useful because when a test fail we often get this kind of message:

```
16:18:05 [py2.7_18.09.1-rc1] =================================== FAILURES ===================================
16:18:05 [py2.7_18.09.1-rc1] _______________________ ExecTest.test_execute_proxy_env ________________________
16:18:05 [py2.7_18.09.1-rc1] tests/integration/api_exec_test.py:28: in test_execute_proxy_env
16:18:05 [py2.7_18.09.1-rc1]     assert exec_log == '''FTP_PROXY=a
16:18:05 [py2.7_18.09.1-rc1] E   AssertionError: assert 'FTP_PROXY=a\n...b\nno_proxy=d\n' == 'FTP_PROXY=a\n...c\nno_proxy=d'
16:18:05 [py2.7_18.09.1-rc1] E       FTP_PROXY=a
16:18:05 [py2.7_18.09.1-rc1] E       HTTPS_PROXY=b
16:18:05 [py2.7_18.09.1-rc1] E       HTTP_PROXY=c
16:18:05 [py2.7_18.09.1-rc1] E       NO_PROXY=d
16:18:05 [py2.7_18.09.1-rc1] E       ftp_proxy=a
16:18:05 [py2.7_18.09.1-rc1] E     + https_proxy=b
16:18:05 [py2.7_18.09.1-rc1] E       http_proxy=c...
16:18:05 [py2.7_18.09.1-rc1] E     
16:18:05 [py2.7_18.09.1-rc1] E     ...Full output truncated (5 lines hidden), use '-vv' to show
16:18:05 [py2.7_18.09.1-rc1] =========================== short test summary info ============================
```

Then I have to edit the Makefile to add `-vv` and re-run the tests.